Dockerfile.ci.test

```diff
@@ -0,0 +1,18 @@
+# This image is used for testing OpenShift Console plugin template capabilities.
+
+FROM quay.io/cajieh0/nodejs:16 AS build
```

Comment:
@cyril-ui-developer We shouldn't have images referencing your personal quay repository and should probably revert #16. Can you describe why that was needed?

Why do we need a separate image for testing? It's much better if we can test the actual Dockerfile that is used to build plugin images, otherwise changes to that Dockerfile can break plugins.